pkgs/servers/owncast/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildGoModule, fetchFromGitHub, bash, which, ffmpeg, makeWrapper, ... }:
```

Comment:
Missing from `all-packages.nix`. Fixed in https://github.com/NixOS/nixpkgs/commit/2e3220d65faaaa236a6ca1916f9ba95d6467fa80.